pkg/api/customization/app/app.go

```diff
@@ -223,3 +238,15 @@ func (w Wrapper) LinkHandler(apiContext *types.APIContext, next types.RequestHan
 	}
 	return nil
 }
+
+func (w Wrapper) validateRancherVersion(externalID string) error {
+	templateVersionID, _, err := hcommon.ParseExternalID(externalID)
+	if err != nil {
+		return err
+	}
+	template, err := w.TemplateVersionLister.Get(namespace.GlobalNamespace, templateVersionID)
```

Comment:
Is this validation needed only for CatalogTemplateVersion only in the global namespace? CatalogTemplateVersions belonging to ClusterCatalogs and ProjectCatalogs are in the namespaces of the clusters/projects

Reply:
Thanks, updated.